src/extension-provider/createExternalExtensionProvider.test.ts

```diff
@@ -126,6 +127,40 @@ describe('createExternalExtensionProvider', () => {
     expect(results).toBeInstanceOf(StreamProvider);
   });
 
+  it('supports Flask', () => {
+    // `global.chrome.runtime` mock setup by `jest-chrome` in `jest.setup.js`
+    (global.chrome.runtime.connect as any).mockImplementation(() => {
```

Comment:
```suggestion
    (global.chrome.runtime.connect as MockedFunction<typeof global.chrome.runtime.connect>).mockImplementation(() => {
```

Same for the other tests.

Reply:
If we don't cast to any it will complain about the `MockPort` class not matching with types. Seems out of scope imo